### ISSUE TITLE
JIT buffer overflow on OpenBSD 5.0 amd64

### ISSUE THREAD

**Issue by racket-bug-submit** [samth/test-bugs#154]
###### Originally submitted by Lasse <lars.engblom at kimitotelefon dot fi> on: Mon Mar 12 02:52:01 -0400 2012

When trying to build racket on OpenBSD 5.0 amd64 the compilation will crash at this point:

cd gc2; make all
make xsrc/precomp.h
env XFORM_PRECOMP=yes ../racketcgc -cqu ../../../racket/gc2/xform.rkt --setup . --cpp "gcc -E -I./.. -I../../../racket/gc2/../include -pthread    -DMZ_NO_ICONV"  --keep-lines -o xsrc/precomp.h ../../../racket/gc2/precomp.c
JIT buffer overflow: 0x20fa5fa35 [0x20fa5c020,0x20fa5fa30](1)!!
Abort trap (core dumped) 
**\* Error code 134
##### Steps to Reproduce:

``` racket
mkdir build
cd build 
../configure --prefix=/whatever/you/want
make
```
##### Release:

```
5.2.1
```
##### Environment:

```
OpenBSD amd64 / Mozilla/5.0 (X11; OpenBSD amd64; rv:5.0) Gecko/20100101 Firefox/5.0
```
###### This bug was converted from Gnats bug [12627](http://bugs.racket-lang.org/old/12627).

[anon-submit; lars.engblom at kimitotelefon dot fi]

**Comment by racket-bug-submit** [comment]
###### On Mon, 12 Mar 2012 10:24:16 -0400, mflatt at racket-lang dot org assigned this bug to mflatt

A commit by mflatt@racket-lang.org has resolved this report
  http://git.racket-lang.org/plt/commit/420d3e7717

**Comment by racket-bug-submit** [comment]
###### On Mon, 12 Mar 2012 10:24:16 -0400, mflatt at racket-lang dot org wrote:

A commit by mflatt@racket-lang.org has resolved this report: 420d3e7717
